index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#">
+          <img src="images/Union.png" alt="menubar" />
+        </a>
+      </nav>
+    </header>
+    <main>
+      <section class="Port">
+        <div hreh="#" class="Menu"></div>
+        <h1>
+          Hey there!<br />
+          I'm Michael
+        </h1>
+
+        <h2>
+          I'm a Software<br />
+          Developer
+        </h2>
+
+        <p>
+          I can help you build a product,<br />
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </p>
+
+        <ul id="SocMediaHand">
+          <li>
+            <a href="https://www.linkedin.com/in/michael-darkwah-81a039141/"
+              ><ion-icon name="logo-linkedin"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="https://github.com/Mike47ip"
+              ><ion-icon name="logo-github"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="https://twitter.com/Mikepee47"
+              ><ion-icon name="logo-twitter"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="https://web.facebook.com/profile.php?id=100089565852279">
+              <ion-icon name="logo-facebook"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="https://web.facebook.com/profile.php?id=100089565852279">
+              <ion-icon name="logo-medium"></ion-icon>
+            </a>
+          </li>
+        </ul>
+
+        <div class="downarrow">
+          <img src="images/Disabled.png" alt="rockbuttom arrow" />
+        </div>
+      </section>
+    </main>
```

Comment:
- [ ] As mentioned by the [previous reviewer](https://github.com/Mike47ip/mobilePortfolioWeb/pull/1#discussion_r1215376021), please add appropriate space on the left and right sides of the page. This space should be added using `padding` and not margin. See my suggestion below 👇🏾 

```css
.port {
  /* ...Other properties */
    padding: 0 20px;
}

nav {
    /* ...Other properties */
    padding: 0 20px;
}
```